Add documentation about ontologies for modules (#3095)
nf-core/website: sites/docs/src/content/docs/nf-core-tools/modules/lint.md

Run the `nf-core modules lint` command to check modules in the current working directory (pipeline or nf-core/modules clone) against nf-core guidelines.

- Use the `--all` flag to run linting on all modules found.
- Use `--dir <pipeline_dir>` to specify another directory than the current working directory.
- Use the `--fix` flag to fix all possible linting errors:

>   - Write the correct input and output channels and channel structure. This will try to use the information existing in your meta.yml, but correct the structure of the `meta.yml` file if needed. Make sure to check that the added changes are correct and update `description`, `pattern` and `type` accordingly.

Reviewer: do you mean main.nf instead of meta.yml in the second sentence here?

Author: No, meta.yml is correct. I want to say that if you have existing information there, it will reuse it, instead of deleting everyting.

Author: would you rephrase it?

Reviewer: yes, it sounds confusing. maybe 
```suggestion
  - Write the correct input and output channels and channel structure. This will try to reuse the information existing in your `meta.yml`, and only change the structure of the file if needed. Make sure to check that the added changes are correct and update `description`, `pattern` and `type` values accordingly.
```

>   - Add the missing ontologies to files in the `meta.yml`. It will add the proper ontologies based on the specified `pattern`. Make sure to add any missing ontology URLs.

Reviewer: ```suggestion
  - Add the missing ontology entries to the input and output entire in `meta.yml`. It will add the proper ontology entries based on the specified `pattern`. Make sure to add any missing ontology URLs.
```

Reviewer: if i am not wrong is ontology the set term. so saying ontologies would mean you talk about different sets, but all should be from EDAM ontolgoy. but not sure.

Author: ```suggestion
  - Add the missing ontology entries to the input and output entries in `meta.yml`. It will add the proper ontology entries based on the specified `pattern`. Make sure to add any missing ontology URLs.
```

Author: adding a new suggestion because there was a typo 🙈
